### DISCUSSION TITLE
How to download all available audio/subtitle?

### DISCUSSION THREAD

**Discussion by yukianime** [crunchy-labs/crunchy-cli#410] [Q&A]
I want to download an anime with all the audio tracks and subtitles it has at the same time, how do I do it?

**Comment by dok1100** [comment]
crunchy-cli archive -m audio -a all -s all -o "{series_name}.S{season_number}E{episode_number}.{audio}.mkv" https://www.crunchyroll.com/series/xxxxxxxxx/xxxxxxxxx

-m audio
is important because otherwise the video will be downloaded for each audio track

-s all
is not needed because if nothing is specified, all available subtitles will always be downloaded

Everything is also in the README.md :)
